src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -120,15 +120,16 @@ class SideNav extends Component {
       >
         <button
           type="button"
-          aria-describedby="va-sidenav-ul-container"
+          aria-label="Pages in this section"
           className={classNames(
             'medium-screen:vads-u-display--none',
             'va-sidenav-default-trigger',
             'vads-u-color--primary',
           )}
           onClick={this.toggleUlClass}
         >
-          In this section <i className="fa fa-bars" />
+          In this section
+          <i aria-hidden="true" className="fa fa-bars" role="presentation" />
```

Comment:
Thanks, va-vfs-bot! Good catch! 

`<i class="fa fa-bars"></i>'

Should be:
`<i class="fa fa-bars" aria-hidden="true" role="presentation" aria-label="Open secondary navigation"></i>`

The line that says `<i aria-hidden="true" className="fa fa-bars" role="presentation" />` would also benefit from adding the `aria-label="Open secondary navigation"`.

Reply:
I've pushed up the requested change. However, for my edification:

What is the purpose of adding `aria-label="Open secondary navigation"`? Is it to get screen readers to say "Open secondary navigation"?

I tried this locally and so far it isn't working. I think this may be because the icon element isn't focusable. (Should it be?) 

Also, if I understand correctly, we have `aria-hidden="true"` and `role="presentation"` to remove the element from the accessibility API. It's unclear to me why we would ever need both, but I see we do this in several places. 

Finally - wouldn't `aria-label` be ignored, since we have `aria-hidden`?

Reply:
It's really hard to provide feedback to code in these test tubes. Sorry, let's swap this.

```html
<button type="button" aria-describedby="va-sidenav-ul-container" class="medium-screen:vads-u-display--none va-sidenav-default-trigger vads-u-color--primary">
  <span class="sr-only">View sub-navigation for </span>
  In this section <i class="fa fa-bars" aria-hidden="true" role="img"></i>
</button>
```

The goal is to make it clear that the button opens the sidenav for those who cannot see.

Reply:
OK thanks for the suggestion!

I added the span but removed the `aria-describedby `and kept the React `onClick` and `className` attributes.

This seems to achieve the desired result. VoiceOver reads "View sub-navigation for In this Section", but does not read the sub-items.

Reply:
I'll check when the work is in staging, but unless y'all changed the code about how the expand/collapse happens, I am sure it still reads the sub-items, if you do control+option+a.